pkg/binfmt/Makefile

```diff
@@ -13,3 +13,11 @@ tag: $(DEPS)
 push: tag
 	DOCKER_CONTENT_TRUST=1 docker pull $(ORG)/$(IMAGE):$(HASH) || \
 	DOCKER_CONTENT_TRUST=1 docker push $(ORG)/$(IMAGE):$(HASH)
+
+check:
+	@if curl -sSL https://registry.hub.docker.com/v2/repositories/$(ORG)/$(IMAGE)/tags/$(HASH) | grep -q 'Not found'; then\
```

Comment:
yes I didn't use `pull` because there is no `pull --dry-run` to check if it's available and signed but not *actually* pull it.

> Especially as we want to test it exists and is signed anyway
Do we want to test that it's signed? See comment below for further thoughts on this.